cmd/icingadb/main.go

```diff
@@ -242,7 +247,17 @@ func run() int {
 						})
 
 						g.Go(func() error {
+							defer close(initConfigSyncDone)
+							defer func() { initConfigSyncDone <- struct{}{} }()
```

Comment:
I would suggest merging those in one single `defer func`, as they belong together. Furthermore, the order matters and reading a stack backwards may lead to confusions.